### ISSUE TITLE
Covert checks and uses of envvars to use the centralized mechanism

### ISSUE THREAD

**Issue by jdharvey-ibm** [carbon-design-system/carbon-platform#202]
Definitely not where we want to do these, haha. I'm basically just waiting on the stuff you have on your branch to make it in and we can do a pass through all of these and make them use the new way of orchestrating them.

_Originally posted by @jdharvey-ibm in https://github.com/carbon-design-system/carbon-platform/pull/191#discussion_r790020264_

**Comment by jdharvey-ibm** [comment]
These conversions have been completed and are merged into the main branch.
